### PR TITLE
Fix Part of #2833 : Add missing named argument in ProfileChooserFragmentTest

### PR DESCRIPTION
**Pull request by mrwhoknows55**
## Explanation
Fixes Part of #2833 : Add missing named argument in ProfileChooserFragmentTest

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt [hunk 1]
```diff
@@ -296,28 +296,28 @@ class ProfileChooserFragmentTest {
     profileTestHelper.initializeProfiles()
     launch(ProfileChooserActivity::class.java).use {
       testCoroutineDispatchers.runCurrent()
-      onView(atPosition(R.id.profile_recycler_view, 0)).perform(click())
+      onView(atPosition(recyclerViewId = R.id.profile_recycler_view, position = 0)).perform(click())
```
Comment (FareesHussain) [inline app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt]:
```suggestion
      onView(atPosition(
        recyclerViewId = R.id.profile_recycler_view,
        position = 0
      )).perform(click())
```

Similarly for `testProfileChooserFragment_clickAddProfile_opensAdminAuthActivity`

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@mrwhoknows55 Thanks for the PR, There are some minor changes left.

Please fill the [contributors survery](https://github.com/oppia/oppia-android/wiki#onboarding-instructions) to get assigned to an onBoarding mentor.

Also please update the PR description
`Fix part of #2833` -> `Fixes part of #2833`  (According to 2nd point in the [checklist](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#checklist-before-submitting-a-pr))

As per the 2nd point in the [instructions](https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change)
Please make sure that your branch has all small characters separated by `-`. Something like `fix-named-args`. You won't be able to do this now but please follow this convention in future PRs.

File: app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt (hunk 1 above)
Comment (mrwhoknows55) [reply app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt]: I have updated these changes

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**